superduperdb/base/build.py

```diff
@@ -140,15 +140,41 @@ def build_datalayer(cfg=None, databackend=None, **kwargs) -> Datalayer:
         logging.error("Error initializing to DataBackend Client:", str(e))
         sys.exit(1)
 
+    # Connect to metadata store.
+    # ------------------------------
+    # 1. try to connect to the metadata store specified in the configuration.
+    # 2. if that fails, try to connect to the data backend engine.
+    # 3. if that fails, try to connect to the data backend uri.
```

Comment:
When the engines of ibis and sqlalchemy are incompatible, directly use the URL to build the metadatastore